app/views/development_metrics/merge_time/_main_metric.erb

```diff
@@ -9,3 +9,11 @@
               suffix: " pull requests",
               metric: merge_time[:per_project_distribution] || merge_time[:per_department_distribution]
             } if merge_time && (merge_time[:per_project_distribution] || merge_time[:per_department_distribution]) %>
+ <%if current_page?(departments_development_metrics_path)%>
```

Comment:
```suggestion
 <% if current_page?(departments_development_metrics_path) %>
```

Reply:
isn't there a way to implement this without `if`s ?

Reply:
uhm.. we could move this into another partial, but that partial will have the conditional.. since this partial (`_main_metric.erb`) is called from several pages to show the chartkick im not sure how to implement this button without a conditional 😅

Reply:
I might be missing context, but wouldn't it be possible to use `content_for` and `yield` ? https://guides.rubyonrails.org/layouts_and_rendering.html#using-the-content-for-method

something like:
the departments_development_metrics page defines `content_for` and this shared view calls `yield :[some name]`